### DISCUSSION TITLE
Options --except=create,show,... and --only=index,store,... in make:controller --resource

### DISCUSSION THREAD

**Discussion by beaubus** [laravel/framework#39581] [Ideas]
A lot of time I use Partial Resource Routes:

```php
Route::resource('invoices', InvoiceController::class)->except(['create', 'show']);
Route::resource('tags', TagsController::class)->only(['index', 'store', 'destroy']);
```

And every time I manually remove unused methods from the controller. 

There is already --api flag, which removes 'create' and 'edit'. 

Would it be cool to have --except and --only options or is it only me, who find it time-saving?

```
php artisan make:controller InvoiceController --resource --except=create,show
php artisan make:controller TagController --resource --only=index,store,destroy
```

**Comment by ordago** [comment]
>There is already --api flag, which removes 'create' and 'edit'.

Technically it doesn't remove them. It uses a different stub.

https://github.com/laravel/framework/blob/5539cf271bc62c5970e845f8bd93362a816d8815/src/Illuminate/Routing/Console/ControllerMakeCommand.php#L40-L60
